library/src/test/resources/pyPfbOutput/showSchema/minimal_data.json

```diff
@@ -1,296 +1 @@
-        "name": "md5sum",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "doc": "The GUID of the object in the index service.",
-        "default": null,
-        "name": "object_id",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "created_datetime",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "ga4gh_drs_uri",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "participant_id",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "specimen_id",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "experimental_strategy",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "study_registration",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "study_id",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "project_id",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": "validated",
-        "name": "state",
-        "type": [
-          {
-            "type": "enum",
-            "name": "submitted_aligned_reads_submitted_aligned_reads_state_0",
-            "symbols": [
-              "validated",
-              "submitted",
-              "released"
-            ]
-          },
-          {
-            "type": "enum",
-            "name": "submitted_aligned_reads_submitted_aligned_reads_state_1",
-            "symbols": [
-              "uploading",
-              "uploaded",
-              "md5summing",
-              "md5summed",
-              "validating",
-              "error",
-              "invalid",
-              "suppressed",
-              "redacted",
-              "live"
-            ]
-          },
-          "null"
-        ]
-      },
-      {
-        "doc": "A project-specific identifier for a node. This property is the calling card/nickname/alias for a unit of submission. It can be used in place of the UUID for identifying or recalling a node.\n",
-        "default": null,
-        "name": "submitter_id",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "subject_id",
-        "type": [
-          "null",
-          "string"
-        ]
-      },
-      {
-        "default": null,
-        "name": "updated_datetime",
-        "type": [
-          "null",
-          "string"
-        ]
-      }
-    ]
-  }
```

Comment:
im confused - did this file have both formatted and nonformatted versions in one?

Reply:
Hah, I didn't notice this.  Yes that's what it appears was in here.  Very confusing.  I think this was a mistake.